src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -526,7 +585,63 @@ public bool Equals(Symbol? member1, Symbol? member2)
                 }
             }
 
-            return !_considerTypeConstraints || HaveSameConstraints(member1, typeMap1, member2, typeMap2);
+            return !_considerTypeConstraints || HaveSameConstraints(member1, typeMap1, member2, typeMap2, ignoreNullability: (_typeComparison & TypeCompareKind.IgnoreNullableModifiersForReferenceTypes) != 0);
```

Comment:
>(_typeComparison & TypeCompareKind.IgnoreNullableModifiersForReferenceTypes) != 0 [](http://example.com/codeflow?start=125&length=81)

This value makes sense, however, I am curious whether any of the existing comparers might change behavior now.

Reply:
None of the existing comparers used `considerTypeConstraints` (they all use `false`)